### ISSUE TITLE
Certificate not actually available?

### ISSUE THREAD

**Issue by Xavantex** [Athozs/hass-additional-ca#10]
**Describe the issue**

I have followed the steps for manual installation, it seems from the logs it is actually loading the certificate.
However, both when I try to use curl and when I look into the various locations/culprits I do not find my self-signed cert anywhere.

**Describe your setup (please complete the following information):**

 - Installation type: 
 HAOS
 - HAOS version (if applicable): 
 13.1
 - Home Assistant core version: 
 2024.9.2
 - Additional CA integration version: 
 0.2.4
 - Kind of certificate you use: 
 Self-signed by vault ed25519 based.

**YAML configuration extract**

Add here an extract of your YAML configuration:

```yaml
default_config:
additional_ca:
  root_ca: root_ca.crt
```

**Logs**
```
2024-09-24 00:23:40.466 INFO (MainThread) [homeassistant.setup] Setting up default_config
2024-09-24 00:23:40.467 INFO (MainThread) [homeassistant.setup] Setup of domain default_config took 0.00 seconds
2024-09-24 00:23:40.510 INFO (MainThread) [custom_components.additional_ca] root_ca (root_ca.crt) -> loaded.
2024-09-24 00:23:40.510 INFO (MainThread) [custom_components.additional_ca] Installation type = Home Assistant OS
2024-09-24 00:23:40.621 INFO (MainThread) [custom_components.additional_ca] Certifi bundle CA ready.
2024-09-24 00:23:40.657 INFO (MainThread) [custom_components.additional_ca] root_ca (root_ca.crt) -> loaded into Certifi CA bundle.
2024-09-24 00:23:40.658 INFO (MainThread) [homeassistant.setup] Setup of domain additional_ca took 1.14 seconds
```

It could be error logs from Home Assistant Settings > System > Logs.

**Additional context**

I have looked in the listed certifi location which does not appear to exist as a directory, I looked in both `/etc/ssl/certs/ca-certificates.crt` and `/usr/local/share/ca-certificates/` none which had my self-signed certificate.

 - What service/device are you trying to reach from Home Assistant with TLS/SSL ?

I am trying to use curl from ssh, and later on the idea is to use it through the shell_command integration.
Need to use uacme to update certificates for https on home assistant, but the acme server is of course behind it's own root certificate.

**Comment by Xavantex** [comment]
Hmm, might be some confusion. Either I am bad at communicating or severely misunderstand HAOS.
I am using an ssh addon from which I can access home assistant core, it is from there that I followed the manual installation steps you have laid out. I put the tar files into `config/custom_components` and changed the `configuration.yaml`.
It is also from core that I am trying to run curl, not in the HAOS. What I can tell from the logs it doesn't seem to complain, but can't find the certificate, neither from soft reboot or hard reboot, have tried power cycling once. The logs provided are the only ones I found under additional_ca ke.
